sechub-shared-kernel/src/test/java/com/mercedesbenz/sechub/sharedkernel/configuration/SecHubConfigurationTest.java

```diff
@@ -522,4 +524,22 @@ public void webscan_empty_includes_excludes() {
         assertTrue("excludes are empty", excludes.get().isEmpty());
         assertEquals(expectedExcludes, excludes.get());
     }
+
+    @Test
+    public void license_scan_test() {
```

Comment:
Please rename, so meaning of test becomes clearer:
```suggestion
    public void a_sechub_configuration_JSON_with_license_scan_can_be_read_and_license_scan_has_correct_data_configuration_reference) {
```